### ISSUE TITLE
Training Scripts

### ISSUE THREAD

**Issue by Rishit-dagli** [swz30/MPRNet#3]
Hi @swz30 ,
I was interested in exploring this a bit further and was wondering if you had any plans to release the training scripts?

**Comment by swz30** [comment]
Hi @Rishit-dagli 

We are aiming to release training codes by the end of February.

**Comment by Rishit-dagli** [comment]
Thanks for addressing this @swz30 , I'll keep a look out for them :)

**Comment by universewill** [comment]
@swz30 Is the training code ready now?

**Comment by howardgriffin** [comment]
Nice work! Waiting for the train code...

**Comment by swz30** [comment]
Hi all,

Training codes are released.

Thank you.

**Comment by ak-karimzai** [comment]
@swz30 can you share plz loss dependency, psnr and ssim value for each epoch?
And the gpu model which you used?
